packages/dx-chart-core/src/plugins/axis/computeds.ts

```diff
@@ -41,14 +46,40 @@ const createVerticalOptions = (position, tickSize, indentFromAxis) => {
   };
 };
 
-export const axisCoordinates = ({
+type AxisCoordinatesArg = {
```

Comment:
I suggest we place all custom defined types and interfaces in separate 'types.ts' files - thus a code will be a bit easier to read and if we want to see a type definition we can use intelliSense/F12

Reply:
done